lib/src/multipart_request.dart

```diff
@@ -83,7 +83,7 @@ class MultipartRequest extends BaseRequest {
   ByteStream finalize() {
     // TODO(nweiz): freeze fields and files
     var boundary = _boundaryString();
-    headers['content-type'] = 'multipart/form-data; boundary="$boundary"';
+    headers['content-type'] = 'multipart/form-data; boundary=$boundary';
```

Comment:
I don't want to go alphanumeric-only unless we absolutely have to, but I've removed all the characters that aren't valid in MIME tokens.